.travis.yml

```
@@ -5,6 +5,7 @@ php:
   - 7.1
   - 7.2
   - 7.3
+  - 7.4snapshot
```

7.4 is going into the final stages.. I guess we can change this to 7.4 now

```suggestion
  - 7.4
```